### ISSUE TITLE
[nation] Join/leave transactions is not saved persistently

### ISSUE THREAD

**Issue by seland** [Bit-Nation/BITNATION-Pangea-libs#62]
## Problem / Task
When joinNation/leaveNation is called not with actual Realm object, transaction is not persisted in Realm.

## Expected Output
Transaction to be saved into database.

## Reproduce
Call joinNation with any object of NationType type, that is not saved in Realm.

## Fix notes
To fix that, Realm object should be fetched firstly using id from nation object passed to joinNation. Then all changes in write should be performed with that object.

**Comment by florianlenz** [comment]
It actually is saved. 

look [here](https://github.com/Bit-Nation/BITNATION-Pangea-libs/blob/release/0.3.2/src/ethereum/nation.js#L310) and specially [here](https://github.com/Bit-Nation/BITNATION-Pangea-libs/blob/release/0.3.2/src/ethereum/nation.js#L324). 

It's also working in the test's ..

**Comment by seland** [comment]
@florianlenz Yes, but if nation that passed to joinNation as parameter is not a Realm object, then it will only be saved on that object, not in the database.
